edb/schema/types.py

```diff
@@ -902,8 +983,16 @@ def create_shell(
         typemods: Any = None,
         name: Optional[str] = None,
     ) -> ArrayTypeShell:
+        if not typemods:
```

Comment:
Do we even use typemods for arrays? Isn't it easier to remove the parameter?

Reply:
I'd thought about cleaning this up later. Simply removing this would break typing, so I decided not to bother for now.